src/CommandingOfficers/RuinedCommander.java

```diff
@@ -0,0 +1,436 @@
+          + "You have a CO zone that covers the area around your COU, and grants bonuses to your units inside.\n"
+          + "The zone expands 1 tile when your meter is half full, and again at full charge.\n"
+          + "All units get +10 attack and +10 Damage Division (DD) while in the zone.\n"
+          + "Your CO may also have another zone boost that stacks with that boost, but may only apply to some unit types.\n"
+          + "The zone becomes global while your power is active, but activating your power requires your COU's action.\n"
+          + "Zone boosts apply to other COs in a tag, even when tagged out.\n"
+          + "CO Energy gained reflects HP damage you deal, and not on the type of unit you hit.\n"
+          + "You can only gain CO energy for combat where the initiator is inside the zone.\n"
+          + "  (This means an enemy in your zone attacking outside your zone can give you charge from your counterattack)\n"
+          + "However, your CO ability does not increase in cost as you use it.\n"
+          + "Veterancy:\n"
+          + "On making a kill, your unit will level up.\n"
+          + "There are 3 veterancy levels; their stat boosts are +5/0, +10/0, and +20/20 attack/DD.\n"
+          + "Your COU is always max level.\n"
+          + "Veterancy bonuses do not apply if this CO is tagged out, but kills are always tracked.\n"
+          );
+  public static final int DOR_STAR_VALUE = 10000;
+  public static final int CHARGERATIO_HP =  1000; // Funds value of 1 HP damage dealt, for the purpose of power charge
+
+  public static final int GENERIC_STAT_BUFF = 10;
+  /** If the unit type matches any flag in this mask, it receives my zone stat boosts */
+  public long canBoostMask = Long.MAX_VALUE;
+  public int zonePow;
+  public int zoneDef;
+  public final int zoneBaseRadius;
+  public int zoneRadius;
+  public boolean zoneIsGlobal = false;
+  ZoneBoostMod myZoneBoost;
+
+  @Override
+  public int getCOUCount() {return 1;}
+
+  public RuinedCommander(int radius, int atk, int def, CommanderInfo info, GameScenario.GameRules rules)
+  {
+    super(info, rules);
+    zonePow = atk;
+    zoneDef = def;
+    zoneBaseRadius = radius;
+    zoneRadius = radius;
+    deployCostPercent = 50;
+    luck = 11;
+  }
+
+  @Override
+  public void initForGame(GameInstance game)
+  {
+    super.initForGame(game);
+    killCounts = StateTracker.instance(game, KillCountsTracker.class);
+    myZoneBoost = new ZoneBoostMod(this);
+    this.army.addUnitModifier(myZoneBoost);
+  }
+  @Override
+  public void deInitForGame(GameInstance game)
+  {
+    super.deInitForGame(game);
+    this.army.removeUnitModifier(myZoneBoost);
+  }
+
+  // Tell MapController and friends that we never have abilities ready, since those go through the COU
+  @Override
+  public ArrayList<CommanderAbility> getReadyAbilities()
+  {
+    return new ArrayList<CommanderAbility>();
+  }
+  @Override
+  public void onCOULost(Unit minion)
+  {
+    modifyAbilityStars(-42);
```

Comment:
Okay, but hear me out: what if one day we have a Commander with an ability that requires more than 42 stars?

Reply:
Then he will be the chosen one.